Model/Utilities/WebKitHandler.swift

```diff
@@ -16,19 +16,14 @@
 import os
 import WebKit
 
-/// Skipping handling for HTTP 206 Partial Content
-/// For video playback, WebKit makes a large amount of requests with small byte range (e.g. 8 bytes)
-/// to retrieve content of the video.
-/// As a result of the large volume of small requests, CPU usage will be very high,
-/// which can result in app or webpage frozen.
-/// To mitigate, opting for the less "broken" behavior of ignoring Range header
-/// until WebKit behavior is changed.
```

Comment:
I did found the root cause of this behaviour. In the original PR the byte range was one off, eg:
```
Byte-Range: 0-5
Content-Length: 5
```
Actually it should be either:
```
Byte-Range: 0-4
Content-Length: 5
```
OR
```
Byte-Range: 0-5
Content-Length: 6
```

Due to this one off error the browser was "kept" downloading the end of the range in chunks about ~8 bytes long.
Once the range was corrected, this side effect no longer exists.